travis-ymls/observatorium.travis.yml

```diff
@@ -0,0 +1,25 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : observatorium
+# Source Repo         : https://github.com/observatorium/observatorium
+# Travis Job Link     : https://travis-ci.com/github/asellappen/observatorium/
+# Created travis.yml  : Yes
+# Maintainer          : Arumugam N S <asellappen@yahoo.com> / Priya Seth<sethp@us.ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+language: go
+
+arch:
+ - amd64
+ - ppc64le
+  
+go:
+ - 1.15
+ - 1.14
+
+script:
+ - export PATH=${PATH}:`go env GOPATH`/bin
+ - go build -v ./...
+ - go test -v ./...
```

Comment:
Apart from these unit tests, there are integrations tests configured through test/integration.sh. Is it possible to run them ?

Reply:
i don t think the scripts  under test folder integration.sh,load.sh and run-local.sh are effectively used and working ,I tried  integration.sh and running for long ,not returning. 

I checked the current ci circleci  pipelines at   https://app.circleci.com/pipelines/github/observatorium/observatorium
and the test integration.sh is not  being done .I have seen only  build, unit test and lint checks are done there.

now i have included lint check also for amd64 alone(ppc64le not supporting due missing dep) in our travis build.

pl review.

Reply:
In this circleci links I see some additional tests ending with "Logs Read/Write tests: OK  "